src/crimson/common/interruptible_future.h

```diff
@@ -513,6 +523,10 @@ struct interruptible_errorator {
       InterruptCond, typename Errorator::template future<>>(
 	Errorator::now());
   }
+
+  using pass_further = typename Errorator::pass_further;
+  using ready_future_marker = ready_future_marker;
```

Comment:
just wanted to ensure that i understand the intention here: so you think it's more handy to have them in `interruptible_errorator`?

Reply:
Yep, this way it matches a normal errorator and I don't have to swap around the code.